README.md

```diff
@@ -1,46 +1,50 @@
 # requests-mauth
 
+[![Build Status](https://travis-ci.org/mdsol/requests-mauth.svg?branch=master)](https://travis-ci.org/mdsol/requests-mauth.svg?branch=master)
+
 requests-mauth is a python requests (http://docs.python-requests.org/en/latest/) Authentication implementation
 for Medidata's _MAuth_ authentication system.
 
-## Pre-requisites ##
+## Prerequisites
 
 To use MAuth authentication you will need:
 
 * An MAuth APP ID
 * An MAuth private key (with the public key registered with Medidata's MAuth server)
 
 
-## Using ##
+## Using
+
+```python
+import requests
+from requests_mauth import MAuth
 
-    import requests
-    from requests_mauth import MAuth
+# MAuth configuration
+APP_UUID = "<MAUTH_APP_UUID>"
+private_key = open("private.key", "r").read()
+mauth = MAuth(APP_UUID, private_key)
 
-    APP_UUID = '55dc88ec-c109-11e1-84f6-1231381b7d70'
-    private_key = open("private.key","r").read()
+# Call an MAuth protected resource, in this case an iMedidata API
+# listing the studies for a particular user
+user_uuid = "10ac3b0e-9fe2-11df-a531-12313900d531"
+url = "https://innovate.imedidata.com/api/v2/users/%s/studies.json" % user_uuid
 
-    mauth = MAuth(APP_UUID, private_key)
+# Make the requests call, passing the auth client
+result = requests.get(url, auth=mauth)
 
-    
-    # Call an MAuth protected resource, in this case an iMedidata API 
-    # listing the studies for a particular user
-    user_uuid = '10ac3b0e-9fe2-11df-a531-12313900d531'
-    url = "https://innovate.imedidata.com/api/v2/users/%s/studies.json" % user_uuid
+# Print results
+if result.status_code == 200:
+    print([r["uuid"] for r in result.json()["studies"]])
+print(result.text)
+```
 
-    # Make the requests call, passing the auth client
-    result = requests.get(url, auth=mauth)
-    
-    # Print results
-    if result.status_code == 200:
-        print([r['uuid'] for r in result.json()['studies']])
-    print(result.text)
 
-Development
------------
-We use [travis](https://travisci.com) for automated CI of the code (and status checks are required to pass prior to PR merges being accepted).  
+## Development
+
+We use [travis](https://travisci.com) for automated CI of the code (and status checks are required to pass prior to PR merges being accepted).
```

Comment:
<img width="513" alt="screen shot 2019-03-05 at 15 35 58" src="https://user-images.githubusercontent.com/16286071/53785598-67a63e00-3f5c-11e9-8f5f-ac2d458b8a5f.png">

Need to update this URL to `https://travis-ci.com`